test/Microsoft.AspNetCore.SignalR.Client.Tests/ServerSentEventsTransportTests.cs

```diff
@@ -65,7 +65,7 @@ public class ServerSentEventsTransportTests
             }
         }
 
-        [Fact]
+        [Fact(Skip = "Flaky tests keep failing")]
```

Comment:
#928 ? Add a link to the skip message if so

Reply:
Also, re-open it, because it's supposedly closed.

Reply:
I'm not checking this in